doc/neps/nep-0030-duck-array-protocol.rst

```diff
@@ -33,30 +33,35 @@ For the purpose above, NEP 22 introduced the concept of duck typing to NumPy
 arrays. The suggested solution described in the NEP allows libraries to avoid
 coercion of a NumPy-like array to a pure NumPy array where necessary, while
 still allowing that NumPy-like array libraries that do not wish to implement
-the protocol to coerce arrays to a pure Numpy array via ``np.asarray``.
+the protocol to coerce arrays to a pure NumPy array via ``np.asarray``.
 
 Usage Guidance
 ~~~~~~~~~~~~~~
 
-Code that uses np.duckarray is meant for supporting other ndarray-like objects
+Code that uses ``np.duckarray()`` is meant for supporting other ndarray-like objects
 that "follow the NumPy API". That is an ill-defined concept at the moment --
 every known library implements the NumPy API only partly, and many deviate
 intentionally in at least some minor ways. This cannot be easily remedied, so
-for users of ``__duckarray__`` we recommend the following strategy: check if the
-NumPy functionality used by the code that follows your use of ``__duckarray__``
+for users of ``np.duckarray`` we recommend the following strategy: check if the
+NumPy functionality used by the code that follows your use of ``np.duckarray`
 is present in Dask, CuPy and Sparse. If so, it's reasonable to expect any duck
 array to work here. If not, we suggest you indicate in your docstring what kinds
 of duck arrays are accepted, or what properties they need to have.
 
+Likewise, if you are implementing an array-like class that provides a
+``__duckarray__`` method, it is important to document what portion(s) of the
+NumPy API is supported.
```

Comment:
Just as a general comment: while I understand that you would like the NEP to be very thorough, I think it's generally unnecessary. IMO, this piece of the NEP is supposed to present guidelines to users (in this case, library developers), rather than tell them how they should manage their projects. I'm not opposed to have this text here, I just think it makes the text unnecessarily longer.

Reply:
agreed, removed again